internal/core/app_deploy.go

```diff
@@ -135,32 +135,34 @@ type deployOperation struct {
 	// Set by init
 	autoHostname pb.UpsertDeploymentRequest_Tristate
 
-	// component is initialized and recycled at various points in the deployment
+	// Component is initialized and recycled at various points in the deployment
 	// operation as we have more templating information for the config.
-	component *Component
+	Component *Component
```

Comment:
I think this change was auto-generated somehow, as I don't see a functional reason for it. Can you revert this?

Reply:
This change was made to be in line with Release. In there the `Component` is exported, I'm not sure why - but I wanted to make it consistent :P